rackspace/compute/v2/servers/results.go

```diff
@@ -0,0 +1,179 @@
+package servers
+
+import (
+	"reflect"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// Server represents a Rackspace server on the user's account.
+type Server struct {
+	// ID uniquely identifies this server amongst all other servers, including those not accessible to the current tenant.
+	ID string
```

Comment:
yeah I agree actually. I think we should decouple the providers as much as possible and not assume openstack is a superset